### ISSUE TITLE
Installation Error (RPi 4)

### ISSUE THREAD

**Issue by yukiqt** [YunoHost-Apps/funkwhale_ynh#220]
## Checksum mismatch.
Full log: https://paste.yunohost.org/raw/ejofetubuv
Important bits:
```
YunoHost was able to download the asset 'api' (https://dev.funkwhale.audio/funkwhale/funkwhale/-/jobs/artifacts/1.2.10/download?job=build_api) for funkwhale, but the asset doesn't match the expected checksum. This could mean that some temporary network failure happened on your server, OR the asset was somehow changed by the upstream maintainer (or a malicious actor?) and YunoHost packagers need to investigate and update the app manifest to reflect this change.
    Expected sha256 checksum: 4a0c0cdd8afc72b3fa9be7019392b3c27d819cf240509234b355231e1c59a11d
    Downloaded sha256 checksum: c0a840f223b4a3a93ffb2639e1bacb709d8ec9ed62214b377971aad5c04ae815
    Downloaded file size: 2.7M
```

**Comment by zakkolar** [comment]
Same issue on  DigitalOcean

https://paste.yunohost.org/raw/ibafecerab

**Comment by yukiqt** [comment]
should be fixed in #221 (not tested on CI yet, Works On My Machine:tm: though)

**Comment by Thovi98** [comment]
@georgkrause Do you have an idea why the checksum of the latest Funkwhale version changes so often?
I’ve already had to change it a couple of times (sometimes to set an old checksum again). Is it due the runners and pipelines issues you mentioned on the [forum](https://forum.funkwhale.audio/d/322-georgs-current-focus-and-aims/2)?

**Comment by yalh76** [comment]
checksum fixed at #222
